superset-frontend/package-lock.json

```diff
@@ -50863,7 +50863,7 @@
       "version": "0.20.3",
       "license": "Apache-2.0",
       "dependencies": {
-        "@types/react-redux": "^7.1.10",
+        "@types/react-redux": "^7.1.34",
```

Comment:
a [desired AFAIK] artifact from `docker-compose up` running `npm i`, I think we should merge this along with any PR